app/models/claim.rb

```diff
@@ -49,6 +49,15 @@ def any_cost_increases?
     core_cost_summary.any_increased?
   end
 
+  def additional_fees
+    totals.fetch(:additional_fees, nil)
+  end
+
+  def additional_fees?
+    FeatureFlags.youth_court_fee.enabled? &&
+      additional_fees&.dig(:total, :claimed_total_inc_vat)&.>(0)
+  end
+
```

Comment:
We should be checking whether claim is elligible for youth court fee (category *A, youth court == yes, nsm, on or after 6th dec)  instead of using total claimed amount